packages/plugins/immediate-introspection/package.json

```diff
@@ -48,7 +48,7 @@
   "typings": "dist/typings/index.d.ts",
   "peerDependencies": {
     "@envelop/core": "^5.0.0",
-    "@sentry/node": "^6 || ^7",
+    "@sentry/node": "^8.0.0",
```

Comment:
Wouldn't this change be considered breaking as v6 and v7 is not supported anymore?